src/main/java/org/isf/admission/model/Admission.java

```diff
@@ -196,8 +197,8 @@ public class Admission extends Auditable<String> implements Comparable<Admission
 	private int lock;                        // default 0
 
 	@NotNull
-	@Column(name = "ADM_DELETED")
-	private String deleted;                    // flag record deleted ; values are 'Y' OR 'N' default is 'N'
+	@Column(name = "ADM_DELETED", columnDefinition = "char(1) default 'N'")
+	private Character deleted = 'N';                // flag record deleted ; values are 'Y' OR 'N' default is 'N'
```

Comment:
```suggestion
	private char deleted = 'N';                // flag record deleted ; values are 'Y' OR 'N' default is 'N'
```

This way there is no boxing/unboxing of the field.    All unit tests pass with this change.